src/engine/renderer/glsl_source/generic_fp.glsl

```diff
@@ -55,4 +56,7 @@ void	main()
 
 	color *= var_Color;
 	outputColor = color;
+#if defined(USE_DEPTH_FLATTEN)
```

Comment:
Could we use a different shader for 2d rendering? I believe adding a new macro to the generic shader will cause 128 additional variants of the shader to have to be compiled. But all of those things are presumably for 3d rendering so there is no need to combine them with this macro.

Reply:
I'm not sure what that would look like. I'm totally a newb with the shader system :(